src/installer/managed/Microsoft.NET.HostModel/AppHost/HostWriter.cs

```diff
@@ -141,6 +142,11 @@ void UpdateResources()
                         throw new Win32Exception(Marshal.GetLastWin32Error(), $"Could not set file permission {filePermissionOctal} for {appHostDestinationFilePath}.");
                     }
                 }
+
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    CodeSign($"-s - {appHostDestinationFilePath}");
```

Comment:
Is this going to work when `appHostDestinationFilePath` has spaces?

Reply:
It will not, although it wouldn't either for the chmod call that we already have. We are relying on the SDK passing a valid apphost destination file path.

Reply:
What is invalid about `/my directory with spaces/app` ? That should work. If it doesn't then it's a bug we should fix.

Reply:
Just tried it -- looks like passing the string through the pinvoke will work just fine but not this. Added quotes here.